scripts/clientconfig.yml

```
@@ -0,0 +1,4 @@
+api:
+  user: broker
+  password: FC_2@8r_LcwufyqQiwc@9TE
```

```suggestion
  password: local-development
```
This looks awfully random to safely put in version control. Since it's for local development, let's use a more obviously non-sensitive value.

---

It was indeed random & generated for local development, but your point is a good one — I will change it